rest_api/index.adoc

```diff
@@ -7,8 +7,171 @@
 :toc: macro
 :toc-title:
 
+toc::[]
+
 The OpenShift distribution of Kubernetes includes the link:./kubernetes_v1.html[Kubernetes v1 REST API]
 and the link:./openshift_v1.html[OpenShift v1 REST API]. These are RESTful APIs accessible via HTTP(s)
 on the OpenShift master servers.
 
-These REST APIs can be used to manage end-user applications, the cluster, and the users of the cluster.
\ No newline at end of file
+These REST APIs can be used to manage end-user applications, the cluster, and the users of the cluster.
+
+== Authentication
+
+API calls must be authenticated with an access token or X.509 certificate. See
+link:../architecture/additional_concepts/authentication.html#api-authentication[authentication architecture]
+for an overview.
+
+This section highlights the token authentication method. With token
+authentication a bearer token must be passed in as an
+link:https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.8[HTTP Authorization header].
+There are two types of access tokens, session and service account.
+
+A **session** token is short-lived, expiring within 24 hours by default. It
+represents a user. The session token may be obtained with this command:
+
+----
+$ oc whoami -t
```

Comment:
Should ```oc login``` first. In the 80% case they are either not logged in or their login token has expired.